shell/platform/tizen/flutter_tizen_view.h

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_FLUTTER_TIZEN_VIEW_H_
+#define EMBEDDER_FLUTTER_TIZEN_VIEW_H_
+
+#include <memory>
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/plugin_registrar.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/tizen/channels/platform_channel.h"
+#include "flutter/shell/platform/tizen/channels/text_input_channel.h"
+#include "flutter/shell/platform/tizen/channels/window_channel.h"
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+#include "flutter/shell/platform/tizen/tizen_window.h"
+
+namespace flutter {
+
+class FlutterTizenView {
+ public:
+  FlutterTizenView(std::unique_ptr<TizenWindow> flutter_tizen_window);
+
+  ~FlutterTizenView();
+
+  // Configures the window instance with an instance of a running Flutter
+  // engine.
+  void SetFlutterTizenEngine(
+      std::unique_ptr<FlutterTizenEngine> flutter_tizen_engine);
+
+  FlutterTizenEngine* flutter_tizen_engine() {
+    return flutter_tizen_engine_.get();
+  }
+
+  TizenWindow* flutter_tizen_window() { return flutter_tizen_window_.get(); }
+
+  // Creates rendering surface for Flutter engine to draw into.
+  // Should be called before calling FlutterEngineRun using this view.
+  void CreateRenderSurface();
+
+  // Destroys current rendering surface if one has been allocated.
+  void DestroyRenderSurface();
+
+  // Callbacks for clearing context, settings context and swapping buffers,
+  // these are typically called on an engine-controlled (non-platform) thread.
+  bool OnMakeCurrent();
+
+  bool OnClearCurrent();
+
+  bool OnMakeResourceCurrent();
+
+  bool OnPresent();
```

Comment:
You may remove the blank lines if the comment in the above is relevant to all the 4 methods.